src/utils/memberListMapper.ts

```diff
@@ -0,0 +1,18 @@
+import { Member } from "@/types";
```

Comment:
We do not need this file at all. If this isn't reusable, it doesn't make sense to keep it in separate file as utility.